syz-cluster/series-tracker/main.go

```diff
@@ -170,6 +183,32 @@ func (sf *SeriesFetcher) handleSeries(ctx context.Context, series *lore.Series,
 	return nil
 }
 
+type seriesProcessor struct {
+	emails map[string]struct{}
```

Comment:
What's the probability to see other fields here?
`type seriesProcessor map[string]struct{}` may simplify the usage.